integration/go.mod

```diff
@@ -3,26 +3,30 @@ module github.com/openshift/osd-network-verifier/integration
 go 1.19
 
 require (
-	github.com/aws/aws-sdk-go-v2 v1.16.16
-	github.com/aws/aws-sdk-go-v2/config v1.17.8
-	github.com/aws/aws-sdk-go-v2/service/ec2 v1.62.0
-	github.com/aws/aws-sdk-go-v2/service/networkfirewall v1.19.1
-	github.com/aws/aws-sdk-go-v2/service/resourcegroupstaggingapi v1.13.19
-	github.com/aws/smithy-go v1.13.3
+	github.com/aws/aws-sdk-go-v2 v1.17.6
+	github.com/aws/aws-sdk-go-v2/config v1.18.18
+	github.com/aws/aws-sdk-go-v2/service/ec2 v1.90.0
+	github.com/aws/aws-sdk-go-v2/service/networkfirewall v1.24.6
+	github.com/aws/aws-sdk-go-v2/service/resourcegroupstaggingapi v1.14.6
+	github.com/aws/smithy-go v1.13.5
 	github.com/jmespath/go-jmespath v0.4.0
-	github.com/openshift/osd-network-verifier v0.0.0-20221004143407-ffba9f3c9174
+	github.com/openshift/osd-network-verifier v0.1.1-0.20230307184731-c061a2224398
 )
 
+replace github.com/openshift/osd-network-verifier => ../
```

Comment:
would this cause conflict with the actual [`go.mod`?](https://github.com/openshift/osd-network-verifier/blob/main/go.mod)

Reply:
Ah this go.mod is separate from the main repo's go.mod, so no conflict.

The purpose of this replace is so that "building/running" the integration test would automatically pick up the code that exists locally vs pinned to an existing commit. I'm good to remove this replace too as well though

Reply:
ack, that explains a lot, imo better to keep it then